aimon/decorators/evaluate.py

```diff
@@ -276,287 +274,22 @@ def evaluate(
             if ag not in record:
                 raise ValueError("Dataset record must contain the column '{}' as specified in the 'headers'"
                                     " argument in the decorator".format(ag))
-        
-        if "context_docs" not in record:
-            raise ValueError("Dataset record must contain the column 'context_docs'")
 
-        _context = record['context_docs'] if isinstance(record['context_docs'], list) else [record['context_docs']]
         # Construct the payload for the analysis
         payload = {
+            **record,
+            "config": config,
             "application_id": am_app.id,
             "version": am_app.version,
-            "context_docs": [d for d in _context],
             "evaluation_id": am_eval.id,
             "evaluation_run_id": eval_run.id,
         }
-        if "prompt" in record and record["prompt"]:
```

Comment:
when async is True, we still need to handle the validation but we can do that in the backend service

Reply:
looks like we have this handled in the backend 👍